lib/ingres_sa_dialect/ingresdbi.py

```diff
@@ -8,6 +8,7 @@
 http://esd.ingres.com.
 """
 from ingres_sa_dialect.base import IngresDialect
```

Comment:
not part of this change but best-practice to move our import below third party imports - do at the same time as this change while here.

See https://peps.python.org/pep-0008/#imports

Semi-related check out:

  * black (very opinionated code formatter)
  * isort (I've only used this a few times in anger, I was very impressed with it)
  * pyflakes (or some other linting tool)

Reply:
Rearranged import statements in accordance with https://peps.python.org/pep-0008/#imports